erpnext/hooks.py

```diff
@@ -17,7 +17,8 @@
 
 app_include_js = "assets/js/erpnext.min.js"
 app_include_css = [
-	"assets/css/erpnext.css"
+	"assets/css/erpnext.css",
+	"assets/erpnext/css/order_desk.css"
```

Comment:
@nehasacher143  Why we need this here. It's available in build.json.

Reply:
@vishdha I have removed it from build.json